tests/integration/ceph_mgr_test.go

```diff
@@ -113,9 +113,28 @@ func (s *CephMgrSuite) TearDownSuite() {
 	s.installer.UninstallRook()
 }
 
-func (s *CephMgrSuite) execute(command []string) (error, string) {
-	orchCommand := append([]string{"orch"}, command...)
-	return s.installer.Execute("ceph", orchCommand, s.namespace)
+func (s *CephMgrSuite) executeWithRetry(command []string, maxRetries int) (string, error) {
+	tries := 0
+	orchestratorCommand := append([]string{"orch"}, command...)
+	for {
+		err, output := s.installer.Execute("ceph", orchestratorCommand, s.namespace)
+		tries++
+		if err != nil  {
+			if maxRetries == 1 {
```

Comment:
where do we pass `maxRetries` equal to 1? It seems that we should always retry up to 3 times.

Reply:
We use` maxRetries=1` when we use directly the method `execute()`. See lines 174/192  in the modified code

Reply:
I don't see any `execute()` method

Reply:
Why is `maxRetries == 1` a special case? Seems like it would be the same as `tries == maxRetries`.

Reply:
> I don't see any `execute()` method

Line 140 of the file "ceph_mgr_test.go" in this pr

Reply:
> Why is `maxRetries == 1` a special case? Seems like it would be the same as `tries == maxRetries`.

If there is no retries ( just one execution) i like more to have just the error message returned by the command for example:
`"blah"`
and not the error message embedded like this:
`"max retries (1) reached, last err: blah"`

It can be done just chnging the message , but I followed your suggestion in https://github.com/rook/rook/pull/8788#discussion_r714119866

Reply:
I see, the different error message is good